setup.cfg

```diff
@@ -1,25 +1,44 @@
 [metadata]
 name = sunkit_image
-author = The SunPy Community
-author_email = sunpy@googlegroups.com
+provides = sunkit_image
 description = A image processing toolbox for Solar Physics.
 long_description = file: README.rst
-license = BSD 3-Clause
-url = http://docs.sunpy.org/projects/sunkit-image/
+long_description_content_type = text/x-rst
+author = The SunPy Community
+author_email = sunpy@googlegroups.com
+license = BSD 2-Clause
+license_file = LICENSE.rst
+url = https://sunpy.org
 edit_on_github = True
 github_project = sunpy/sunkit-image
+platform = any
+keywords = solar physics, solar, science, sun, image analysis
+classifiers =
+  Development Status :: 3 - Alpha
+  Intended Audience :: Science/Research
+  License :: OSI Approved :: BSD License
+  Natural Language :: English
+  Operating System :: OS Independent
+  Programming Language :: Python
+  Programming Language :: Python :: 3
+  Programming Language :: Python :: 3.6
+  Programming Language :: Python :: 3.7
+  Topic :: Scientific/Engineering :: Physics
 
 [options]
 python_requires = >=3.6
 packages = find:
 include_package_data = True
-setup_requires = setuptools_scm
+setup_requires =
+  setuptools_scm
+  cython
```

Comment:
Does this just want to go in here, isn't it better off in `pyproject.toml` (maybe both I am not sure).

Reply:
Should add it to both.